.ci/deps.opam.sh

```diff
@@ -1,13 +1,14 @@
 set -e
 set -x
 
-TERM=dumb
+export TERM=dumb
```

Comment:
unrelated change.

what is this fixing?

create an issue about it.

Reply:
I had faced an error saying TERM not found...
This change solved it...
Will try to find the link to that log

Reply:
ping?  This would be a separate bug...